.github/workflows/pr-build.yaml

```
@@ -237,12 +241,21 @@ jobs:
         env:
           # we hardcode plugins version, so it's predictable in e2e tests
           GORELEASER_CURRENT_TAG: "v0.0.0-latest"
-          OUTPUT_MODE: "archive"
+          OUTPUT_MODE: "binary"
+          SINGLE_PLATFORM: "true"
+          PLUGIN_TARGETS: "kubernetes,kubectl,cm-watcher,echo,helm"
         run: |
           make build-plugins
 
+      - name: CLI Cache
+        uses: actions/cache@v3
+        with:
+          path: dist/botkube-cli_linux_amd64_v1/botkube
```

```suggestion
              path: |
                ~/.cache/go-build
                ~/go/pkg/mod
                dist/botkube-cli_linux_amd64_v1/botkube
```

I would explicity add Go build cache directory in this case.